wizeline/project-lab
# Fix/validate comment

Bug #177 


#### What does this PR do?

Validates that the comment can't be empty when add comment button is pressed

#### Where should the reviewer start?

Go to the project details page.

#### How should this be manually tested?

1. Go to the comments section
2. Press add comment
3. Verify that the validation is thrown.
4. Go to the comments section
5. Press add comment
6. Verify that the validation is thrown.

#### Any background context you want to provide?

Currently when you try to save an empty comment the application executes an action and in the console you can see an error from the backend.

Validate that the comment cannot be empty when the add comment button is pressed
#### What are the relevant tickets?
 #177
